### DISCUSSION TITLE
Exclusion list and polarization

### DISCUSSION THREAD

**Discussion by vwcruzeiro** [TinkerTools/tinker-gpu#184] [Q&A] [answered]
Hello Tinker experts!

We have been reading about the AMOEBA force field, including one of Piquemal's publications: https://pubs.acs.org/doi/10.1021/ct500998q . We have been trying to better understand equation 22 in this paper, more specifically the reason why the induced dipole moments without considering the exclusion list (computed from "direct field") are used in combination with the electric field that contains exclusions ("polarization field"). Furthermore, while we were trying to run some test system using Tinker, we noticed that for two bonded atoms, the induced dipole between them will polarize each other. We are curious about why this design choice was made. We would appreciate it if you could point out a resource that talks about that; maybe a thesis from one of your students or a previous publication we may have not seen.

Thank you so much!

Vinicius and @henryw7

**Comment by jayponder** [comment, marked as answer]
Hi, The short answer to your question is that this is just how we have chosen to design the AMOEBA force field, because it "works". All these issues with exclusion rules, scaling, damping, etc. are model choices made during the design of the force field. You should go back and look at the several earlier papers on the AMOEBA model primarily from my group and Pengyu Ren's group at UT-Austin that contain detail regarding some of these choices.

For example, as you note above, AMOEBA uses a modified "Thole" damping scheme to describe polarization. Under this model, all "mutual" polarization is included, even between directly bonded atoms. Note that it's different for "direct" polarization, where exclusion rules are used to omit directly bonded and other very local interactions.

**Reply by vwcruzeiro** [reply]
Thank you so much for taking the time to answer us @jayponder . All the best!
